src/components/StructuredSeoData.tsx

```diff
@@ -26,9 +25,19 @@ export const StructuredSeoData = ({ uri }: StructuredSeoDataProps) => {
     }
   }, [uri]);
 
-  return seoData ? (
-    <Helmet>
-      <script type="application/ld+json">{seoData}</script>
-    </Helmet>
-  ) : null;
+  useEffect(() => {
```

Comment:
Skulle helst slått sammen til en useEffect, og dermed slippe unna state for dette, men syns det ble mer komplisert kode, og siden denne komponenten bare returnerer `null` så skal det ikke merkes at det blir en rerender heller